src/octoprint/static/js/app/viewmodels/temperature.js

```diff
@@ -200,13 +200,28 @@ $(function () {
                 .heatedChamber.subscribe(self._printerProfileUpdated);
         });
 
+        self.markings = [];
+        self.MARKING_TYPE = {
+            PRINT: "Print",
+            PAUSE: "Pause",
+            RESUME: "Resume",
+            CANCEL: "Cancel"
+        };
+
+        var markingColors = {};
+        markingColors[self.MARKING_TYPE.PRINT] = "#218656";
+        markingColors[self.MARKING_TYPE.PAUSE] = "#FDC02F";
+        markingColors[self.MARKING_TYPE.RESUME] = "#27CAEE";
+        markingColors[self.MARKING_TYPE.CANCEL] = "#DA3749";
```

Comment:
I'd suggest the following here... 

```suggestion
        self.marking_props = {
            "start": gettext("Start"),
            "pause": gettext("Pause"),
            "resume": gettext("Resume"),
            "cancel": gettext("Cancel"),
        }
```

Then switch the backend markings to lower case names (& also add a regular `done` one because that's kinda missing right now) and use them to create an additional css class attached to each label (`tempMarkingLabel-{type}`). Adjust the less file, include the color definitions for the classes in there (because that way any third party themes can also override the colors).

Reply:
Unfortunately, the lines are drawn on the canvas, only the labels are html elements, so I don't believe I'll be able to make the colors CSS customizable 😞

Reply:
Ah, good point. Then leave it with the colors hard coded and I'll see if I can come up with something long term.

Reply:
I _could_ try some hacks like: creating the labels, getting the computed styles for the labels and determine the color for each and then remove the temporary labels

Reply:
Not important enough (for now) to introduce hacks ^^